### DISCUSSION TITLE
Tracking with 1 animal

### DISCUSSION THREAD

**Discussion by Johan154** [talmolab/sleap#1006] [General]
Currently, I am working on a project where I use SLEAP, within this project, only 1 animal is visible per frame. Does tracking help in this situation to increase the accuracy or does this only work for situations with multiple animals within 1 frame? 
Can somebody also explain to me what the idea is behind tracking in general? In the documentation it states that tracking can only be used for predicted frames, but I would suppose that it can also be used for training in order to track a certain key point throughout multiple frames.

**Comment by roomrys** [comment]
Hi @Johan154,

Currently we do not use any classifiers for tracking. At this point in time, we solve the tracking problem by using a cost function where we compare the distances between instances in sequential frames. Tracking is unnecessary for single animal projects and will only create more work with no reward. Linking a relative discussion #987.

Let us know if this helps and/or if you have more questions.

Thanks,
Liezl
